fdbserver/workloads/ConfigureDatabase.actor.cpp

```diff
@@ -462,11 +461,6 @@ struct ConfigureDatabaseWorkload : TestWorkload {
 				wait(success(
 				    IssueConfigurationChange(cx, logTypes[deterministicRandom()->randomInt(0, length)], false)));
 			} else if (randomChoice == 7) {
```

Comment:
why we no longer want to test `backup_worker_enabled` config anymore?

Reply:
Backup Worker noop mode is removed. If we enable backup worker without any backup job, mutations will stay on tlog   because no backup worker is going to pop them..